Building issues

### Steps to Reproduce

1. $ cd xamarin-macios
2.  $ make world

### Expected Behavior
Application build successfull
### Actual Behavior
Application build failed
### Environment
=== Visual Studio Enterprise 2019 for Mac ===

Version 8.1 (build 2742)
Installation UUID: aa3b163a-671b-4fef-86f8-4bf9baca2491
	GTK+ 2.24.23 (Raleigh theme)
	Xamarin.Mac 5.6.0.25 (d16-0 / 50f75273)

	Package version: 600000302

=== Mono Framework MDK ===

Runtime:
	Mono 6.0.0.302 (2019-02/fdf6490d049) (64-bit)
	Package version: 600000302

=== NuGet ===

Version: 5.0.2.5988

=== .NET Core ===

Runtime: /usr/local/share/dotnet/dotnet
Runtime Version: 2.1.11
SDK: /usr/local/share/dotnet/sdk/2.1.700/Sdks
SDK Version: 2.1.700
MSBuild SDKs: /Library/Frameworks/Mono.framework/Versions/6.0.0/lib/mono/msbuild/Current/bin/Sdks

=== Xamarin.Profiler ===

Version: 1.6.10
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

=== Updater ===

Version: 11

=== Apple Developer Tools ===

Xcode 10.2.1 (14490.122)
Build 10E1001

=== Xamarin.Mac ===

Xamarin.Mac not installed. Can't find /Library/Frameworks/Xamarin.Mac.framework/Versions/Current/bin/mmp.

=== Xamarin.iOS ===

Xamarin.iOS not installed.
Can't find mtouch or the Version file at /Library/Frameworks/Xamarin.iOS.framework/Versions/Current.

=== Xamarin Designer ===

Version: 16.1.0.464
Hash: 66bb7b43f
Branch: remotes/origin/d16-1-new-document-model
Build date: 2019-06-07 07:10:32 UTC

=== Xamarin.Android ===

Not Installed

=== Microsoft Mobile OpenJDK ===

Java SDK: Not Found

Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

=== Android Device Manager ===

Version: 1.2.0.44
Hash: aac645b
Branch: remotes/origin/d16-1
Build date: 2019-05-29 19:55:24 UTC

=== Xamarin Inspector ===

Version: 1.4.3
Hash: db27525
Branch: 1.4-release
Build date: Mon, 09 Jul 2018 21:20:18 GMT
Client compatibility: 1

=== Build Information ===

Release ID: 801002742
Git revision: ed27233de1b33083ea2234bc66a3b7824b99bbc7
Build date: 2019-06-12 10:18:51+00
Build branch: release-8.1
Xamarin extensions: 98dc4d7de2257c9b7342dee023910e16d02f8e65

=== Operating System ===

Mac OS X 10.14.5
Darwin 18.6.0 Darwin Kernel Version 18.6.0
    Thu Apr 25 23:16:27 PDT 2019
    root:xnu-4903.261.4~2/RELEASE_X86_64 x86_64



### Build Logs

Making all in builds
Downloading https://xamjenkinsartifact.azureedge.net/mono-sdks/mac-release-Darwin-04659832c3d3bd698f4cef33f41e810f98dd5b61.zip...
Downloading https://xamjenkinsartifact.azureedge.net/mono-sdks/ios-release-Darwin-04659832c3d3bd698f4cef33f41e810f98dd5b61.zip...
Running autogen for ../external/mono
LN       libmono-2.0.dylib
LN       libmono-2.0.a
make[2]: *** [downloads/mac-release-Darwin-04659832c3d3bd698f4cef33f41e810f98dd5b61.zip] Error 22
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [downloads/ios-release-Darwin-04659832c3d3bd698f4cef33f41e810f98dd5b61.zip] Error 22
Ran autogen for ../external/mono in 0m 17s
make[1]: *** [all-recurse] Error 1
make: *** [world] Error 2

--- a/Make.config
+++ b/Make.config
@@ -9,6 +9,9 @@ $(TOP)/Make.config.inc: $(TOP)/Make.config
 	@printf "MAC_COMMIT_DISTANCE:=$(shell LANG=C; export LANG && git --git-dir $(TOP)/.git log `git --git-dir $(TOP)/.git blame -- ./Make.versions HEAD | grep MAC_PACKAGE_VERSION= | sed 's/ .*//' `..HEAD --oneline | wc -l | sed 's/ //g')\n" >> $@
 	@if which ccache > /dev/null 2>&1; then printf "ENABLE_CCACHE=1\nexport CCACHE_BASEDIR=$(abspath $(TOP)/..)\n" >> $@; echo "Found ccache on the system, enabling it"; fi
 	@if test -d $(TOP)/../maccore; then printf "ENABLE_XAMARIN=1\n" >> $@; echo "Detected the maccore repository, automatically enabled the Xamarin build"; fi
+	@# Build from source for Xcode 11 support until mono has packages
+	@# TODO: to be removed when we can use the mono archive again.
+	@printf "MONO_BUILD_FROM_SOURCE=1\n" >> $@; echo "Disabled the packaged mono build, since it's currently not supported in the xcode11 support branch."
 
 include $(TOP)/Make.versions
 

--- a/configure
+++ b/configure
@@ -94,6 +94,10 @@ while test x$1 != x; do
         echo "MONO_BUILD_FROM_SOURCE=1" >> "$CONFIGURED_FILE"
         shift
         ;;
+    --disable-packaged-mono=no | --disable-packaged-mono=false)
+        echo "MONO_BUILD_FROM_SOURCE=" >> "$CONFIGURED_FILE"
+        shift
+        ;;
 	--help|-h)
 		show_help
 		exit 0

--- a/jenkins/build.sh
+++ b/jenkins/build.sh
@@ -113,9 +113,6 @@ if test -z "$ENABLE_DEVICE_BUILD"; then
 	CONFIGURE_FLAGS="$CONFIGURE_FLAGS --disable-ios-device"
 fi
 
-# Build from source for Xcode 11 support until mono has packages
-CONFIGURE_FLAGS="$CONFIGURE_FLAGS --disable-packaged-mono"
-
 make reset
 make git-clean-all
 make print-versions

